pkg/controllers/federatedhpa/federatedhpa_controller.go

```diff
@@ -68,6 +68,7 @@ type FederatedHPAController struct {
 	RESTMapper                meta.RESTMapper
 	EventRecorder             record.EventRecorder
 	TypedInformerManager      typedmanager.MultiClusterInformerManager
+	ClusterCacheSyncTimeout   metav1.Duration
```

Comment:
Nice, 100% agree that we need to use the same configuration for informers inside of a controller.